code/modules/admin/verbs/diagnostics.dm

```diff
@@ -246,4 +246,51 @@
 			str += "\n|{{reagent|[R.result_amount]|[R.result]}}"
 		else
 			str += "\n|''(Check [R.type]/on_reaction()!)''"
-	text2file(str+"\n|}","chemistry-recipes.wiki")
\ No newline at end of file
+	text2file(str+"\n|}","chemistry-recipes.wiki")
+
+// For /vg/ Wiki docs
+/client/proc/dump_chems()
+	set category = "Debug"
+	set name = "Dump Chemicals"
+
+	var/paths = typesof(/datum/reagent) - /datum/reagent
+	var/num = 0
+	var/subtypes = 0
+	var/normal_shc = 0
+	var/normal_density = 0
+	var/str = {"
+{| class="wikitable"
+|-
+! Name
+! Reactants
+! Result"}
+	for(var/path in paths)
+		var/datum/reagent/R = new path()
+		num++
+		str += {"
+|-
+! [num]: [R.name]"}
+		if(R.id)
+			str += "\n|<ul>"
+			str += "\n|<li>{{ID = [R.id]}}</li>"
```

Comment:
I'm not sure you understand how mediawikis work.

This is outputting a table, which is in the form of:
```
{| class="css classes" (<table> start)
|- (<tr> start)
! (<th>) contents
! class="th css classes" | contents
|- (another row starts)
| td cell contents
| td attributes | td contents
|} (<table> ends)
```

Shit like `{{hurr}}` are templates.

https://www.mediawiki.org/wiki/Help:Templates
https://www.mediawiki.org/wiki/Help:Tables

Reply:
was literally just something to help me debug and find what didn't have SHC and density set. Not intended for real use.

Reply:
I've commented it out now

Reply:
If you don't intend to use it again, remove it completely